enterprise/kernel/src/main/java/org/neo4j/kernel/enterprise/builtinprocs/TransactionId.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2002-2017 "Neo Technology,"
+ * Network Engine for Objects in Lund AB [http://neotechnology.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Neo4j is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.neo4j.kernel.enterprise.builtinprocs;
+
+import org.neo4j.kernel.api.KernelTransactionHandle;
+
+public class TransactionId
+{
+    private static final String USER_TRANSACTION_ID_PREFIX = "transaction-";
+
+    private TransactionId()
+    {
+    }
+
+    static String ofTransactionHandle( KernelTransactionHandle transaction )
+    {
+        return USER_TRANSACTION_ID_PREFIX + transaction.getUserTransactionId();
+    }
```

Comment:
Maybe this can be a `name` function or something like that, on the `KernelTransactionHandle`. The name of this class clashes with another one in `kernel.impl.store`.